brevbaker/build.gradle.kts

```diff
@@ -26,6 +26,7 @@ dependencies {
     api(project(":brevbaker-dsl"))
     ksp(project(":template-model-generator"))
     implementation("org.jetbrains.kotlinx:kotlinx-html-jvm:0.12.0")
+    implementation("io.ktor:ktor-client-content-negotiation:$ktorVersion")
```

Comment:
Er ikke `brevbaker` bibliotek-modulen vår? Bør vi ha avhengighet til ktor her?

Reply:
Godt poeng, det bør vi jo unngå. Ser det einaste vi treng det for her er HttpStatusCode og ContentType, og begge er jo enkle å skrive oss bort frå ved litt enkel hardkoding.

Reply:
Oppdatert no, erstatta med hardkoding av det vi bruker frå desse to.